### ISSUE TITLE
New Kotlin sourceset layout breaks Moko resources

### ISSUE THREAD

**Issue by MarkRS-UK** [icerockdev/moko-resources#410]
The new sourceset layout brought on by the Kotlin update to 1.8 seems to break Moko resources.

On update to K1.8, my Android build fails complaining that resources are duplicated.  They aren't.

This can be avoided by sticking with the old sourceset layout, it's a Gradle option, but this will not last forever.

**Comment by msa1422** [comment]
For me, the project compiled fine with K1.8.0, AGP-7.4.0 with warning 
```
W: Duplicate content root detected
Path [../PetsApp/petsearch-shared/resources/build/generated/moko/androidMain/assets] 
of module [PetFinder-Showcase.petsearch-shared.resources.main] was removed from modules [PetFinder-Showcase.petsearch-shared.resources.main]" 
Also 1 more path was deduplicated.
```

But after upgrading AGP to 7.4.1, compilation fails with error
```
// For all resources in MR/assets
AGPBI: {"kind":"error","text":"Duplicate resources","sources":[{"file":{"description":"bg_paw_print_loading.svg","path":"../PetsApp-PetFinder/PetsApp/petsearch-shared/resources/build/generated/moko/androidMain/assets/bg_paw_print_loading.svg"}},{"file":{"description":"bg_paw_print_loading.svg","path":"../PetsApp/petsearch-shared/resources/build/generated/moko/androidMain/assets/bg_paw_print_loading.svg"}}],"tool":"Resource and asset merger"}

Execution failed for task ':petsearch-shared:resources:packageDebugAssets'.
```

Here is the [link to the project](https://github.com/msa1422/KMM-Arch-PetSearch) where issue can be reproduced.

**Comment by Alex009** [comment]
will be released in 0.21.0
